config/initializers/doorkeeper.rb

```diff
@@ -1,3 +1,10 @@
+# Doorkeeper uses ActiveSupport::LazyLoadHooks to lazily load its models.
```

Comment:
This one was fun to find.

Reply:
hahahaah what a weird thing

Reply:
Yes! I thought I was going crazy for a while until I found a random discussion about a similar error in Rake tasks. That led me to `ActiveSupport::LazyLoadHooks` and figuring out that `active_record/base` needs to be required for it to work. I'm jut happy I didn't waste even more time on this.